docker-compose.yml

```diff
@@ -29,3 +29,13 @@ services:
       dockerfile: setupMinio.Dockerfile
     depends_on:
       - minio
+
+  seed_local_data:
```

Comment:
seeds local data into the app. Upserts organization, incentives, and a product catalog for said organization